keyboards/fractal/keymaps/default/keymap.c

```diff
@@ -0,0 +1,225 @@
+/* Copyright 2015-2017 Jack Humbert
```

Comment:
Change the copyright info here.

Reply:
@drashna I think for minor modifications, the existing text is actually more or less preferred.

https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=copyright-blurb